package.json

```diff
@@ -14,6 +14,7 @@
     "@jest/globals": "workspace:*",
     "@jest/test-utils": "workspace:*",
     "@tsconfig/node10": "^1.0.8",
+    "@tsd/typescript": "^4.5.5",
```

Comment:
Can be any lower version too.

Reply:
yeah, let's match the version of typescript we use, then I'll bump this along with TS itself in #11142